src/platform/plugins/shared/workflows_execution_engine/server/plugin.ts

```diff
@@ -125,7 +126,7 @@ export class WorkflowsExecutionEnginePlugin
                 coreStart, // TODO: move to dependencies
                 config,
                 logger,
-                fakeRequest: fakeRequest!,
+                fakeRequest: fakeRequest || ({} as KibanaRequest),
```

Comment:
Instead of pretending we have a request. Wouldn't it make more sense to add a check at the top of the function and fail if that's the case?
```ts
        createTaskRunner: ({ taskInstance, fakeRequest }) => {
          if (!fakeRequest) {
            throw new Error('Cannot run workflow without Kibana Request');
          }
```

Reply:
I think we already have this check. @talboren added it in setupDependencies function.